packages/react-link/src/index.ts

```diff
@@ -1 +1,9 @@
-export * from './Link';
+export {
+  Link,
+  linkClassName,
```

Comment:
There is also [linkClassNames](https://github.com/microsoft/fluentui/blob/master/packages/react-link/src/components/Link/useLinkStyles.ts#L11) now.

Reply:
Added.